assets/panel/debugger.properties

```diff
@@ -19,9 +19,10 @@ collapsePanes=Collapse panes
 copyToClipboard.label=Copy to clipboard
 copyToClipboard.accesskey=C
 
-# LOCALIZATION NOTE (copySource): This is the text that appears in the
+# LOCALIZATION NOTE (copySource.label): This is the text that appears in the
 # context menu to copy the selected source of file open.
 copySource=Copy
```

Comment:
Just noticed while translating: is the copySource string still used anywhere after this PR? If not, it should be removed

Reply:
good call. I can check

Reply:
actually we're close to removing the old debugger. Lets wait till then and then remove a lot of strings

Reply:
Sounds good, thanks (there's another PR with a lot more strings)